config/routes.rb

```diff
@@ -75,6 +75,16 @@
   namespace :admin do
     get 'index', path: '/'
 
+    resources :report_generator do
```

Comment:
I'm not sure this `resources : report_generator` is the way you want to do this. Look at the output of rake routes:

```
report1_admin_report_generator_index GET    /admin/report_generator/report1(.:format)                      admin/report_generator#report1
report2_admin_report_generator_index GET    /admin/report_generator/report2(.:format)                      admin/report_generator#report2
report3_admin_report_generator_index GET    /admin/report_generator/report3(.:format)                      admin/report_generator#report3
                                     POST   /admin/report_generator/report1(.:format)                      admin/report_generator#report1
                                     POST   /admin/report_generator/report2(.:format)                      admin/report_generator#report2
                                     POST   /admin/report_generator/report3(.:format)                      admin/report_generator#report3
        admin_report_generator_index GET    /admin/report_generator(.:format)                              admin/report_generator#index
                                     POST   /admin/report_generator(.:format)                              admin/report_generator#create
          new_admin_report_generator GET    /admin/report_generator/new(.:format)                          admin/report_generator#new
         edit_admin_report_generator GET    /admin/report_generator/:id/edit(.:format)                     admin/report_generator#edit
              admin_report_generator GET    /admin/report_generator/:id(.:format)                          admin/report_generator#show
                                     PATCH  /admin/report_generator/:id(.:format)                          admin/report_generator#update
                                     PUT    /admin/report_generator/:id(.:format)                          admin/report_generator#update
                                     DELETE /admin/report_generator/:id(.:format)                          admin/report_generator#destroy
                  search_admin
```

There's a lot of things in there from the 'classic' 7 routes. You'll have to do some search on how to do this properly. Maybe `namespace :report_generator` or you can exclude actions